libc-test/build.rs

```diff
@@ -653,9 +659,18 @@ fn main() {
     // fails on a lot of platforms.
     let mut cfg = ctest::TestGenerator::new();
     cfg.skip_type(|_| true)
-        .skip_struct(|_| true)
+        .skip_struct(move |name| {
+            match name {
+                // The termios2 struct is the same as `termios` on powerpc, so it's actually
+                // undeclared in the headers. We'll still expose it there to still provide
+                // some type safety, but we can't check it directly.
+                "termios2" if !powerpc => false,
```

Comment:
Could this use `struct_name` to rename the struct to assert `termios2` is the same as `termios` on powerpc?